develop/tutorials/articles/100-tooling/03-blade-cli/05-deploying-projects-with-blade-cli.markdown

```diff
@@ -35,31 +35,42 @@ framework using Felix Gogo shell and deploys the project directly to the OSGi
 container using Felix File Install commands. The command uses the default
 `11311` port by default.
 
-<!--
-You can also specify a custom port to deploy your module to using the `-p`
-parameter followed by the port number. For instance, you could run `blade deploy
--p 8090` to deploy to port 8090.
--->
++$$$
+
+**Note:** The `blade deploy` command requires a Gradle/Maven wrapper to
+successfully execute. To ensure the availability of a build tool wrapper, work
```

Comment:
I don't think we need this sentence if we change blade deploy

Reply:
@gamerson same for this one. Will `blade deploy` be updated by end of week when docs are published?